admin-apis/.env

```diff
@@ -0,0 +1,4 @@
+ACCOUNT_KEY=
+ACCOUNT_SECRET=
+API_ENDPOINT=
```

Comment:
Default value here should be prod endpoint

Reply:
done